### PR TITLE
Fix grey on grey text

### PR DESCRIPTION
**Pull request by ehaynes99**
Fixes dark grey on slightly-less-dark-grey in reviews, Q&A, etc.
E.g. https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint&ssr=false#review-details

### CODE REVIEW

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -403,6 +403,15 @@ INVERT
 
 ================================
 
+marketplace.visualstudio.com
```
Comment (alexanderby) [inline src/config/dynamic-theme-fixes.config]: Please, preserve alphabetical order.

File: src/config/dynamic-theme-fixes.config [hunk 2]
```diff
@@ -403,6 +403,15 @@ INVERT
...
+
+CSS
+.details-tab {
+  color: rgba(201, 199, 197, 0.9);
```
Comment (alexanderby) [inline src/config/dynamic-theme-fixes.config]: Please, check how this fix looks in Light mode too. I've added a pattern for applying settings values to color, it looks like `${rgba(201, 199, 197, 0.9)}`, where the color inside `${}` is a source color without modifications. Also `!important` keyword should be used. So, your fix will look like `color: ${rgba(201, 199, 197, 0.9)} !important;`.

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (ehaynes99) [reply src/config/dynamic-theme-fixes.config]: Oops, thought I did. Off by one...

**Comment by ehaynes99** [comment]
I made the requested changes. Looks good in light mode. Sorry about that...

**Pull request merged**